Wireframe/css/style.css

```diff
@@ -0,0 +1,57 @@
+body{
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 20px;
+    margin: 0;
+    padding: 0 0 80px 0;
+}
+header{
+    text-align: center;
+    padding: 20px;
+    background-color: #f4f1f1;;
+}
+
+.wrapper{
+    padding: 20px;
+    width: 80%;
+    margin: 0 auto;
+}
+
+article:first-child {
+    display: block;
+    margin-bottom: 20px;
+    margin: 0 auto 2rem auto;
+    border: 1px solid #ccc;
+}
+
+article:not(:first-child) {
+    display: inline-block;
+    width: 40%;
+    justify-content:space-between;
+}
+.article2{
+   margin-right: 5rem;
+}
+
+.article3{
+    margin-left: 4rem;
+    
+    
+}
+
+.article-image {
+    width: 100%;
+    height: 330px;
+}
+
+
+footer {
+    position: fixed;         /* Fix the footer to the bottom */
+    bottom: 0;               /* Align it to the bottom of the viewport */
+    width: 100%;             
+    background-color: #f4f1f1;
+    color: rgb(31, 23, 23);            
```

Comment:
Hi Samira, personally, I think it's often simpler to use hexadecimal (hex) values for colours instead of rgb. But this is fine. I find rgba useful for transparent colours. You could have used `1F1717` for this colour.

But what you have done isn't incorrect. Another good reason for using hex colours is that they display more consistently across different machines and browsers i.e. laptops, tablets, mobiles and/or Chrome, MS Edge and so on.